src/api/libcellml/units.h

```diff
@@ -400,6 +400,33 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
 
+    /**
+     * @brief Return a bool which indicates if the units are loosely comparable.
```

Comment:
not a fan of "loosely comparable" - much prefer the brief description used in the python bindings!